frontend/src/views/housing/project/ProjectView.vue

```diff
@@ -0,0 +1,436 @@
+      </span>
+      <span v-else>Navigator: -</span>
+    </div>
+    <div><h3 class="mb-5">Required permits</h3></div>
+    <div
+      v-if="!(permitsNeeded?.length || permitsNotNeeded?.length)"
+      class="empty-block p-5"
+    >
+      We are investigating the permits required for this project.
+    </div>
+
+    <Card
+      v-for="permit in permitsNeeded"
+      :key="permit.permitId"
+      class="app-primary-color permit-card hover-underline cursor-pointer"
+    >
+      <template #content>
+        <h5 class="m-0 p-0">{{ permit.businessDomain }}: {{ permit.name }}</h5>
+      </template>
+    </Card>
+    <Accordion
+      v-if="permitsNotNeeded?.length"
+      class="app-primary-color"
+    >
+      <AccordionTab header="Not needed">
+        <div>
+          We have also investigated the following permits as requested. These permits are not required for this project.
+        </div>
+        <ul class="mt-4 mb-0">
+          <li
+            v-for="permit in permitsNotNeeded"
+            :key="permit.permitId"
+            class="m-0"
+          >
+            {{ permit.businessDomain }}: {{ permit.name }}
+          </li>
+        </ul>
+      </AccordionTab>
+    </Accordion>
+    <h3 class="mt-8 mb-5">Submitted applications</h3>
+    <div
+      v-if="!(permitsSubmitted.length || permitsNotNeeded.length)"
+      class="empty-block p-5"
+    >
+      You will see your permit applications here once submitted.
+    </div>
+    <Card
+      v-for="permit in permitsSubmitted"
+      :key="permit.permitId"
+      class="permit-card"
+    >
+      <template #title>
+        <h5 class="m-0 app-primary-color">{{ permit.businessDomain }}: {{ permit.name }}</h5>
+        <Divider />
+      </template>
+      <template #content>
+        <div class="grid">
+          <div class="col-2">
+            <div :class="['authIndicator', toKebabCase(displayTrackerStatus(permit.authStatus, permit.status))]">
+              {{ displayTrackerStatus(permit.authStatus, permit.status) }}
+            </div>
+          </div>
+          <div class="col-2">
+            <div>{{ permit?.statusLastVerified ? formatDate(permit.statusLastVerified) : '' }}</div>
+            <div :class="['sub-label', { 'm-0': permit?.statusLastVerified }]">Last verified date</div>
+          </div>
+          <div class="col-2">
+            <div>{{ permit?.trackingId ? permit.trackingId : '' }}</div>
+            <div :class="['sub-label', { 'm-0': permit?.trackingId }]">Tracking ID</div>
+          </div>
+          <div class="col-2">
+            <div>{{ permit?.issuedPermitId ? permit.issuedPermitId : '' }}</div>
+            <div :class="['sub-label', { 'm-0': permit?.issuedPermitId }]">Permit ID</div>
+          </div>
+        </div>
+      </template>
+    </Card>
+  </div>
+</template>
+
+<style scoped lang="scss">
+a {
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 2.125rem;
+}
+
+h3 {
+  font-size: 1.75rem;
+}
```

Comment:
Was it requested by the UX team to override the header sizes here?

Reply:
Yes, they wanted to follow the BCGov design guideline stuff here. I suggested we may need to follow up for the other pages, but that it would be another time. Future pages they'll likely want to override too.